client/README.md

```diff
@@ -1,10 +1,8 @@
-<!--
-TODO: Need correct marketplace ID
+
 [![Version](https://vsmarketplacebadge.apphb.com/version/jpogran.Puppet.svg)](https://marketplace.visualstudio.com/items?itemName=jpogran.Puppet)
 [![Installs](https://vsmarketplacebadge.apphb.com/installs-short/jpogran.Puppet.svg)](https://marketplace.visualstudio.com/items?itemName=jpogran.Puppet)
--->
 
-![Puppet Logo](images/Puppet-Logo-Amber-sm.png)
```

Comment:
This is a duplicate of what shows in the gallery, and looks odd when packaged, so removed it